### ISSUE TITLE
Feature request: Option to hide user from home but keep them visible on lists

### ISSUE THREAD

**Issue by MaripozeDoRodo** [Dimillian/IceCubesApp#444]
A option on user profile to hide them from home timeline, but keeping their toots still visible on lists.

**Comment by edel79** [comment]
Agree. Ivory does that very well.

**Comment by rafacampoamor** [comment]
I totally agree. Few instances have the "exception list" activated, so being able to do it from the app, as in Mona or Ivory is an indispensable improvement. 🥺🙏🏻

**Comment by pjorgeromero** [comment]
I would be a great feature. For those accounts you have to follow but want to read at your own pace.

**Comment by jeffsikes** [comment]
This feature exists in the API now as part of the Mastodon 4.2.0 rollout. As of today, [about 50% of active servers](https://fedidb.org/software/mastodon) are on the new version, including Mastodon social. 

It's the `exclusive` flag on the lists endpoint for Create and Update, true = hide from timelines, only show in feed.

[https://docs.joinmastodon.org/methods/lists/#create](https://docs.joinmastodon.org/methods/lists/#create)

A few apps have added this already, but does require checking for support of the feature before displaying.

<img width="574" alt="image" src="https://github.com/Dimillian/IceCubesApp/assets/6627582/d3dbd21e-6f65-4eff-a441-b127544e55db">
